apps/connection/src/connection.repository.ts

```diff
@@ -316,4 +316,52 @@ export class ConnectionRepository {
       throw error;
     }
   }
+
+  async deleteConnectionRecordsByOrgId(orgId: string): Promise<IDeletedConnectionsRecord> {
+    const tablesToCheck = ['credentials', 'presentations'];
```

Comment:
Please create an enum for the table name, it may break when the table will migrated

Reply:
Resolved